wagtail/admin/templates/wagtailadmin/userbar/item_page_add.html

```diff
@@ -3,7 +3,7 @@
 
 {% block item_content %}
     <div class="wagtail-action">
-        <a href="{% url 'wagtailadmin_pages:add_subpage' self.page.id %}" target="_parent">
+        <a href="{% url 'wagtailadmin_pages:add_subpage' self.page.id %}" target="_parent" role="menuitem">
```

Comment:
@Stormheg I wonder if the `div.wagtail-action` wrapper is needed anymore? `.wagtail-action` only sets a transparent background. I am all for simplified markup, where possible

Reply:
Interesting! I'll have a look and see if I can remove it :slightly_smiling_face:

Reply:
Removed it :+1: